backend/structs.py

```diff
@@ -26,7 +27,7 @@ def is_authenticated(
 
 @dataclass
 class WebsocketInfo:
-    pk: int  # pylint: diable
+    pk: int  # pylint: disable
```

Comment:
```suggestion
    pk: int
```

Reply:
i played with this last night, turns out this line doesnt have a pylint error anyway.  must have had something different initially?  idr why i put it there haha